spec/support/request_helpers.rb

```diff
@@ -89,23 +89,11 @@ def check_400_on_invalid_json
     end
   end
 
-  def check_content_type_header
```

Comment:
Based on [this commit](https://github.com/alphagov/publishing-api/commit/4a63a91fc95d61463894988e34d2649f7ef6d3d9) it looks like we were doing it to better handle Rails HTML errors, see [this ticket](https://trello.com/c/8zCaPXNh/96-improve-url-arbiter-response-handling-in-publishing-api).  It may be that this is no longer relevant?